templates/articles/_comment.html

```diff
@@ -0,0 +1,27 @@
+<turbo-frame id="comment-{{ comment.pk }}">
+<div class="card">
+    <div class="card-block">
+        <p class="card-text">
+            {{ comment.body }}
+        </p>
+    </div>
+    <div class="card-footer">
+        <a href="" class="comment-author">
+            <img
+                    src="{{ comment.author.img }}"
+                    class="comment-author-img"
+            />
+        </a>
+        &nbsp;
+        <a href="" class="comment-author">{{ comment.author }}</a>
```

Comment:
Does this work? If so, neat. I'd been using something like `comment.author.user.username`

Reply:
I think that works because we have a `__str__` method in our Profile model:
![Screen Shot 2021-01-08 at 10 50 30](https://user-images.githubusercontent.com/1490875/104022664-5a34ed00-519f-11eb-80dc-3eec2dbf719a.png)


We also have a missing link:
```suggestion
        <a href="{% url 'profile_view' comment.author.user.username %}" class="comment-author">{{ comment.author }}</a>
```

We could implement the `get_absolute_url` to our Profile, avoiding a lot of those repetitions across our templates.

Reply:
Yea I saw that as well - the profile also has a name method, but not of the generated users have first or last names so it returns an empty string, It should probably be updated to return first + last and if missing the username.